docs/home.md

```diff
@@ -1,12 +1,31 @@
 # Overview of TechPass
 
-## What is TechPass?
-TechPass is the Identity & Access Management (IAM) and Single Sign-on (SSO) solution to access [Singapore Government Technology Stack (SGTS) services](#sgts-services). It provides a seamless login experience across SGTS services and allows centralised user access control.
+TechPass is the Identity & Access Management (IAM) and Single Sign-on (SSO) solution for the following [Singapore Government Technology Stack (SGTS) services](#sgts-services-integrated-with-techpass).
 
-### SGTS services
+<a id="sgts-services-integrated-with-techpass">
+
+**SGTS services integrated with TechPass**
+
+</a>
 
 Following are the list of SGTS services that uses TechPass as their IAM service.
 
+- Container Stack(CStack): CStack is a cloud-based container development and hosting platform.
+- Docs portal: Your hub for all the technical documentation such as developer guides, API references and tutorials.
+- GCC Common Services: 
+- GCC 2.0: A "wrapper" platform that provides government agencies with a consistent means to adopt commercial cloud solutions offered by cloud service providers.
+- SEED: SEED is the Singapore Government's implementation of IAM and zero trust platform to protect Government engineering resources from unauthorised access.
+- SHIP-HATS: The Continuous Integration/Continuous Delivery CI/CD component in SGTS with security and governance guardrails that enables developers to plan, build, test and deploy code to production.
+- TechBiz: One-stop portal to subscribe to SGTS services.
+
+For more information about these services, see [SG Government Tech Stack Collection](https://www.developer.tech.gov.sg/products/collections/singapore-government-tech-stack/)
+
+<!--
```

Comment:
u might want to remove this comment

Reply:
you mean the explanation i wrote there, or the comment text itself?

I have removed the explanation.